maven-javadoc-plugin/src/main/java/org/apache/maven/plugin/javadoc/AbstractJavadocMojo.java

```diff
@@ -288,6 +289,8 @@
      */
     private static final float SINCE_JAVADOC_1_8 = 1.8f;
 
+    private static final float JAVA_VERSION_FLOAT = JavadocUtil.parseJavadocVersion(SystemUtils.JAVA_VERSION);
```

Comment:
@Tunaki refactoring to `JavaVersion` would also have been my preference but that's tricky for two reasons
1. `AbstractJavadocMojo.version` is Maven container injected, I don't know how we could customise the injection process [Mapping Complex Objects](https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Mapping_Complex_Objects) does not seem to be a solution
1. Unfortunately both the parsing code and the code for accessing the parsed float is not public in commons-lang3. We ultimately need the parsed float in `AbstractJavadocMojo.addStandardDocletOptions`.

Initially I copy and pasted the parsing code from commons-lang 2.6 but that was quite a bit of code.